src/audio/emscripten/SDL_emscriptenaudio.c

```diff
@@ -155,11 +155,8 @@ static bool EMSCRIPTENAUDIO_OpenDevice(SDL_AudioDevice *device)
             Module['SDL3'] = {};
         }
         var SDL3 = Module['SDL3'];
-        if (!$0) {
-            SDL3.audio_playback = {};
-        } else {
-            SDL3.audio_recording = {};
-        }
+        SDL3.audio_playback = {};
+        SDL3.audio_recording = {};
```

Comment:
Seems harmless enough, although I can't see how this could relate to LTO.

One thing I notice is that you removed the reference to `$0` which means you no longer need to pass the `device->recording` param on line 174